### PR TITLE
Add duplicate product docs

### PR DESCRIPTION
**Pull request by octaedro**
### Changes proposed in this Pull Request:

The WooCommerce PR https://github.com/woocommerce/woocommerce/pull/46141 added a new REST API endpoint meant to `duplicate a product`.

This PR adds the documentation necessary for that new endpoint

It adds the documentation under `v3` and under `wp-api-v3`.

### How to test the changes in this Pull Request:

1. Checkout this branch and build the project
```
sh build.sh
```
2. After running the build process, you will see a `build` folder in your project.
3. Go to the `build` folder and open the following files in your browser:
- `./index.html`
- ~`./v3.html`~ (updated @mattsherman - our change is only for the WP API, not the legacy API)
4. Go to `Product > Duplicate product` in the sidebar.
5. Verify that the documentation looks correct.

### CODE REVIEW

File: source/includes/v3/_products.md [hunk 1]
```diff
@@ -1629,6 +1629,115 @@ woocommerce.get("products").parsed_response
 	<code>tag</code>, <code>shipping_class</code> and <code>pa_*</code> filters are available starting from WooCommerce 2.5.
 </aside>
 
+## Duplicate product ##
```
Comment (mattsherman) [inline source/includes/v3/_products.md]: This documentation is for the *legacy* WC API. As such, we shouldn't be updating the doc for this, as our change is for the non-legacy WP REST API.

**Review by mattsherman — CHANGES_REQUESTED** [review]
Looks good except that we shouldn't be updating the legacy API docs too.

Ping me when that is removed and I'll re-review/approve!

File: source/includes/v3/_products.md (hunk 1 above)
Comment (octaedro) [reply source/includes/v3/_products.md]: Done in the commit [b8345d7](https://github.com/woocommerce/woocommerce-rest-api-docs/pull/244/commits/b8345d7d191191b038a087404aadea51d30a2799)

Comment (octaedro) [reply source/includes/v3/_products.md]: @mattsherman Could you take another look at this PR?

**Review by mattsherman — APPROVED** [review]
Thanks for adding this documentation @octaedro !

**Pull request merged**